Automattic/pocket-casts-android
# Migrate player artwork section to compose

## Description
This migrates player artwork section to Compose.


## Testing Instructions
1. Enable "Use episode artwork" from the Appearance settings
2. Play episodes from the list below
3. ✅  Notice that artwork is rendered properly
3. ✅  Notice that tapping on chapter URL (when present) opens the URL 

Square Image
10. Devil in the Detail: https://pca.st/episode/f53b1287-2231-4715-8e87-c2316e35f2b8

Rectangle Image
From Twit events audio: https://pca.st/episode/e703a4b2-9d5f-4da2-b9bd-32ca2dae6f11

Video 
From Ted Talks HD: https://pca.st/episode/8b958a43-5e8e-4b39-a57b-6ef2991b32a6

With Chapter Url 
From Changelog: https://pca.st/episode/edfd7342-cea6-49b5-8583-930c6066f869


## Screenshots or Screencast 

Phone Portrait

Normal | Rectangle | Video | Small Phone Square | Small Phone Rectangle
----|---|---|---|---
<img width=200 src="https://github.com/user-attachments/assets/efb4e88a-51fa-440f-ba5a-dc541573e8ab"/>| <img width=200 src="https://github.com/user-attachments/assets/01f83d79-0710-43c7-b5b3-94108db7bb31"/>| <img width=200 src="https://github.com/user-attachments/assets/7b135f4d-8a27-4782-8099-8bb88e42c5fd"/>| <img width=200 src="https://github.com/user-attachments/assets/e72ca73d-a72e-4f8a-8c7b-68951d3b44fe"/>| <img width=200 src="https://github.com/user-attachments/assets/e24376f4-863f-481e-a8d1-baa000cef817"/>

Phone Landscape | Phone Landscape Video
---|---
![phone_lanscape](https://github.com/user-attachments/assets/1cb2c0d6-e232-415d-b4be-f5cd8ef102f6) | ![phone_landscape_video](https://github.com/user-attachments/assets/abc7c845-1b7a-41d7-8536-5ab347fd6ebe)


Tablet Portrait with Chapter Link | Tablet Portrait Rectangle | Tablet Portrait | Tablet Landscape Rectangle
--|--|--|--
![tablet_port](https://github.com/user-attachments/assets/39ca1490-c558-4a50-bed3-688184bbbcc4)| ![tablet_port_recta](https://github.com/user-attachments/assets/f6333a81-a303-4357-bfbe-fb76f93eba23) | ![tablet_port](https://github.com/user-attachments/assets/9630a411-702d-4e4a-b5c1-7e253c29f8b5) | ![tablet_land](https://github.com/user-attachments/assets/b9fd529b-3b7b-4ea9-a07a-537f1e2a4934)


## Checklist
 
#### I have tested any UI changes...
